src/util/string_expr.h

```diff
@@ -24,17 +24,15 @@ binary_relation_exprt length_ge(const T &lhs, const exprt &rhs)
   return binary_relation_exprt(lhs.length(), ID_ge, rhs);
 }
 
-template <typename T>
-binary_relation_exprt length_gt(const T &lhs, const exprt &rhs)
+inline binary_relation_exprt greater_than(const exprt &lhs, const exprt &rhs)
```

Comment:
pass expressions by copy and use `std::move` :mushroom:

Reply:
done